pages/index.tsx

```diff
@@ -0,0 +1,68 @@
+import Image from 'next/image'
+import Layout from '../components/Layout'
+import styles from '../styles/Home.module.css'
+import epfl from '../assets/img/logos/logo-sponsor-epfl.svg'
+import trust from '../assets/img/logos/logo-sponsor-trust-valley.svg'
+import vaud from '../assets/img/logos/logo-sponsor-spei.svg'
+import illustration from '../assets/img/cds-illustration.png'
+
+export default function Home() {      
+  return (   
+    <div className='flex flex-col w-full bg-red'>
+      <div className='absolute top-10 right-0 overflow-hidden max-h-[85%] invisible md:visible'>
+          <Image src={illustration} />  
+      </div>      
+      <div className='w-full justify-center items-center max-w-[85%]'>
+        <p className={`m-5 mt-5 sm:p-10 sm:text-xl text-lg ${styles['main-header-title']}`}>Condensation is an <strong>open-source</strong> project disrupting how we create applications to match the challenges of the era of data.</p>
```

Comment:
If the text of this line is not long enough, the background width is wrong and the image is absent.